pkgs/applications/version-management/git-and-tools/stgit/default.nix

```diff
@@ -14,7 +14,8 @@ stdenv.mkDerivation {
     sha256 = "0xpvs5fa50rrvl2c8naha1nblk5ip2mgg63a9srqqxfx6z8qmrfz";
   };
 
-  buildInputs = [ python2 git ];
+  nativeBuildInputs = [ git ];
+  buildInputs = [ python3 ];
```

Comment:
```suggestion
```

the builder does this and will always implicate the proper python version

Reply:
done